### PR TITLE
docs: Edit user text for "write down your xxx" for mnemonic generation #92

### PR DESCRIPTION
**Pull request by barriebyron**
**:praise: ** such great progress team! 

@wal33d006 I took a peek at https://github.com/tendermint/flutter/pull/29 to see the awesomeness happen. 

I was curious about what the user was being asked to write down (the password? or the mnemonic?). I took a first attempt at clarifying the language for writing down (whatever it is we should write down).

Questions and feedback most welcome!

### CODE REVIEW

File: lib/views/mnemonic_onboarding.dart [hunk 1]
```diff
@@ -44,7 +44,7 @@ class _MnemonicOnboardingState extends State<MnemonicOnboarding> {
                       ),
                       SizedBox(height: 16),
                       Text(
-                        'Please write this down somewhere safe, if you lose this, you will not get your account back. Not even Demeris could help you with that.',
+                        'Follow best practices for security. Be sure to write your mnemonic pass phrase in a safe place. This phrase is the only way to recover your account if you forget your password. Without your password or recovery passphrase, account recovery is not possible. Even Emeris cannot help you.',
```
Comment (barriebyron) [inline lib/views/mnemonic_onboarding.dart:47]: @wal33d006 let's be clear about what we are asking the user to write down 
ambiguous pronoun references happen all the time, especially in code comments and technical docs 

see https://developers.google.com/style/pronouns 
Key Point: Ensure that a pronoun clearly refers to its antecedent (the noun that it's replacing).

https://developers.google.com/style/word-list#letter-t
the "this" in "please write this down" was unclear to me (my editor calls this an unclear reference, (our docs have a lot of those "opportunities" for clarity)

this, that
Where possible, put a noun after this or that for clarity. If doing so results in clunky prose, then don't do it; but even then, try thinking about what the noun would be. If you aren't sure what noun this or that refers to, then consider rephrasing.

Comment (wal33d006) [reply lib/views/mnemonic_onboarding.dart:47]: Awesome! That's amazing observation! Thanks for the update!

**Review by wal33d006 — APPROVED** [review]
⚡️

File: lib/views/mnemonic_onboarding.dart (hunk 1 above)
Comment (barriebyron) [reply lib/views/mnemonic_onboarding.dart:46]: @wal33d006 so my guess was correct? we're asking the user to write down their mnemonic pass phrase?

Comment (wal33d006) [reply lib/views/mnemonic_onboarding.dart:46]: @barriebyron 
Correct. We are asking the user to write this mnemonic passphrase in a secure place

**Comment by barriebyron** [comment]
oh I see you approved thanks @wal33d006 I don't seem to have merge permissions, please merge on my behalf, thank yoU!

**Pull request merged**